content/W2016/W16-good-shit.md

```diff
@@ -0,0 +1,22 @@
+Title: Yee
```

Comment:
Why did you add me to this repo dammit Fatema
On Mar 15, 2016 7:03 PM, "Fatema Boxwala" <notifications@github.com> wrote:

> In content/W2016/W16-good-shit.md
> <https://github.com/wics-uw/git-workshop-W16/pull/1#discussion_r56256280>:
>
> > @@ -0,0 +1,22 @@
> > +Title: Yee
>
> "Yee" is not a great name for an event :)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/wics-uw/git-workshop-W16/pull/1/files/693edc0d35ffca919c73e3a8ce1b8d351b354570#r56256280>
>